inst/tutorials/vaccines/vaccines.Rmd

```diff
@@ -0,0 +1,332 @@
+    clinical_incidence_rendering_max_ages = 5 * year,
+    individual_mosquitoes = FALSE
+  )
+)
+
+simparams <- set_equilibrium(parameters = simparams, init_EIR = starting_EIR)
+
+# Run a model with no interventions in a setting with no seasonality
+output_control <- run_simulation(timesteps = sim_length * 2, parameters = simparams)
+```
+
+## More exercises: Vaccination
+
+Now that you have been introduced to malariasimulation, we will review a few vaccine use cases that might be helpful. This tutorial is largely based on the malariasimulation vaccine [vignette](https://mrc-ide.github.io/malariasimulation/articles/Vaccines.html)-- refer to this for more practice! 
+
+We can use malariasimulation to implement a theoretical transmission blocking vaccine (TBV) or a pre-erythrocytic vaccine. Vaccines can be administered using either a mass vaccination strategy or an age-based strategy. 
+
+To carry this out, we can utilize the functions `set_pev_epi()` or `set_mass_pev().` For the purposes of this training, we will focus on the administration of a pre-erythrocytic vaccine (IE, RTS,S or R21).
+
+First, let's look at the vaccine efficacy parameters for RTS,S. These are saved within malariasimulation:
+
+```{r efficacy_profile, exercise = TRUE}
+
+malariasimulation::rtss_profile # Parameters for the first three doses of RTS,S
+
+malariasimulation::rtss_booster_profile # Parameters for the fourth (booster) dose of RTS,S
+```
+
+These parameters are used to model the relationship between antibody titre and vaccine efficacy, based on previous modelling work conducted in the [group](https://pubmed.ncbi.nlm.nih.gov/37080831/). The meaning of these parameters can be found [here](https://mrc-ide.github.io/malariasimulation/reference/create_pev_profile.html) -- you will likely not need to modify them directly, but know that these parameters impact the magnitude and duration of vaccine impact. 
+
+```{r quiz1, echo= FALSE}
+
+quiz(caption = "Vaccine Efficacy Question",
+     question_numeric(
+       "What is the vaccine efficacy of the first three doses of the RTS,S vaccine?",
+       answer(0.93, correct = TRUE), allow_retry = T, correct = "Correct! This means that the maximum efficacy against infection is 93%.")
+     )
+
+```
+
+
+Now, let's obtain a set of parameters and run a baseline model. We will use `set_equilibrium()` to intialise the model at a certain EIR.
+
+```{r baseline_model, exercise = TRUE}
+
+
+sim_length <- 3 * year
+human_population <- 10000
+starting_EIR <- 20
+
+simparams <- get_parameters(list(
+    human_population = human_population,
+    clinical_incidence_rendering_min_ages = 0,
+    clinical_incidence_rendering_max_ages = 5 * year,
+    individual_mosquitoes = FALSE
+  )
+)
+
+simparams <- set_equilibrium(parameters = simparams, init_EIR = starting_EIR)
+
+# Run a model with no interventions in a setting with no seasonality
+output_control <- run_simulation(timesteps = sim_length * 2, parameters = simparams)
+
```

Comment:
I think we need to show somthing has happened here

```suggestion

# Let's take a look at the model output
head(output_control)
```

Reply:
Have added an incidence plot